packages/stats-py/openpls_driver.py

```
@@ -0,0 +1,131 @@
+        from importlib.metadata import version
+
+        return version("openpls-engine")
+    except Exception:
+        return "unknown"
+
+
+def main() -> None:
+    try:
+        multiprocessing.set_start_method("fork", force=True)
+    except (RuntimeError, ValueError):
+        pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(RuntimeError, ValueError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(RuntimeError, ValueError): ...`

(SIM105)